core/src/main/java/com/netflix/conductor/metrics/WorkflowMonitor.java

```diff
@@ -77,48 +77,57 @@ public WorkflowMonitor(
             initialDelayString = "${conductor.workflow-monitor.stats.initial-delay:120000}",
             fixedDelayString = "${conductor.workflow-monitor.stats.delay:60000}")
     public void reportMetrics() {
-        try {
-            if (refreshCounter <= 0) {
+        if (refreshCounter <= 0) {
                 workflowDefs = metadataService.getWorkflowDefs();
                 taskDefs = new ArrayList<>(metadataService.getTaskDefs());
                 refreshCounter = metadataRefreshInterval;
-            }
-
-            getPendingWorkflowToOwnerAppMap(workflowDefs)
-                    .forEach(
-                            (workflowName, ownerApp) -> {
-                                long count =
-                                        executionDAOFacade.getPendingWorkflowCount(workflowName);
-                                Monitors.recordRunningWorkflows(count, workflowName, ownerApp);
-                            });
-
-            taskDefs.forEach(
-                    taskDef -> {
+        }
+
+        // This getPendingWorkflowCount is not implemented in scylla execution DAO class. 
+        // Disabling this temporarily.
+        // try {
+        //         getPendingWorkflowToOwnerAppMap(workflowDefs)
+        //                 .forEach(
+        //                         (workflowName, ownerApp) -> {
```

Comment:
have added this in staging_debug PR along with its refresh config. Please ignore this,

Reply:
done